src/emittermixin.js

```diff
@@ -357,6 +367,48 @@ const EmitterMixin = {
 				destinations.delete( emitter );
 			}
 		}
+	},
+
+	/**
+	 * Returns an `EmitterMixin` instance with given `id`, which this emitter is listening to. Returns `null` if
+	 * this emitter do not listen to any emitter identified by `id`.
+	 *
+	 * @protected
+	 * @param {String} id Unique emitter id.
+	 * @returns {EmitterMixin|null}
+	 */
+	_getEmitterListenedTo( id ) {
+		if ( this[ _listeningTo ] && this[ _listeningTo ][ id ] ) {
+			return this[ _listeningTo ][ id ].emitter;
+		}
+
+		return null;
+	},
+
+	/**
+	 * Emitter's unique id.
+	 *
+	 * **Note:** `_emitterId` can be set only once.
```

Comment:
KK, here's explanation why we need the setter now: https://github.com/ckeditor/ckeditor5-utils/issues/120#issuecomment-271881520. Let's simplify this in the followup.